android/src/main/java/com/brentvatne/common/api/ControlsConfig.kt

```diff
@@ -7,6 +7,8 @@ class ControlsConfig {
     var hideSeekBar: Boolean = false
     var seekIncrementMS: Int = 10000
     var hideDuration: Boolean = false
+    var showSubtitleButton: Boolean = true
```

Comment:
@seyedmostafahasani Can you please start by extracting showSubtitleButton & showSettingButton in a separated PR please ?
I am not sure this PR will me merged unfortunnatly, I am afraid of posssible regressions :/

Reply:
Thanks for reviewing again.
Yeah, sure.
I will do it.